pydantic_ai_slim/pydantic_ai/models/__init__.py

```diff
@@ -382,6 +390,16 @@ async def request(
         """Make a request to the model."""
         raise NotImplementedError()
 
+    async def count_tokens(
+        self,
+        messages: list[ModelMessage],
+        model_settings: ModelSettings | None,
+        model_request_parameters: ModelRequestParameters,
+    ) -> BaseCountTokensResponse:
```

Comment:
Can this directly return `Usage` instead of needing `BaseCountTokensResponse.to_usage`?